.github/workflows/checkproto.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Set up Cocos-AI
         run: |
           # Rename .pb.go files to .pb.go.tmp to prevent conflicts
-          for p in $(ls manager/*.pb.go); do
+          for p in $(ls pkg/manager/*.pb.go); do
```

Comment:
I do not like that this in `pkg` is called `manager`, like other `manager` in the top level. It is confusing.